### DISCUSSION TITLE
Path Split

### DISCUSSION THREAD

**Discussion by ntmanxp** [netbox-community/netbox#11770] [Q&A]
Hello
I'm trying to model and trace connection between 2 switches and several Fiber Panels.
Some switches uses SFP modules ( bidirectional in some cases, that single fiber do the connection ) and in that case, the single fiber is traced of through the devices.
But when the SFP is splitted in 2 fibers ( Tx/Rx ) and this plugged to a front port of the patch panel, then I got a Path Split, and no way to continue the path.
I modeled with Splices in the rear port; Also with single cable rear to rear, and then rear to front in 1:1 relationship in each side, but no way. Can't make it work!!

See this:

![imagen](https://user-images.githubusercontent.com/25610170/219456283-05bb6528-6a0e-47d1-a2f6-3d5a42425a37.png)

![imagen](https://user-images.githubusercontent.com/25610170/219456649-6cf67fa9-ec48-4b5d-aa77-fdf29ee5cdbf.png)

The mirror side is:
![imagen](https://user-images.githubusercontent.com/25610170/219456428-5f0a9f1b-03ff-4536-931b-26791903ae15.png)

![imagen](https://user-images.githubusercontent.com/25610170/219456750-0ee44c2b-03aa-418f-81b1-a535a19977e4.png)

![imagen](https://user-images.githubusercontent.com/25610170/219457699-427fc444-2b0b-4d08-b5f1-f9acb0f43f33.png)

And if I do click on the node ( any, Rear Port r9 or r10 ) I got 'Not Path Found'. Am I misunderstanding the model??
Most of the Fiber Patch panels, uses a Fiber Cable of 12/24 or 48 strands, and are connected back to back from one building/room to the other. It's not any kind of MPO. 

Please any advice should be really appreciated.

Andres.-

**Comment by ntmanxp** [comment]
Hi
Continuing the thread, if I define rear ports like splices, between Fiber Panels, but with discrete patchs between middle Fiber Panels ( or ODFs ) the path split came again.
Tried one model this way:

![imagen](https://user-images.githubusercontent.com/25610170/220647546-d18cc64b-f387-414c-9834-b4ce59625bbe.png)

And got:
![imagen](https://user-images.githubusercontent.com/25610170/220647469-48f46396-d8e9-45cd-9997-a8873b3e46ba.png)

And in the other side:
![imagen](https://user-images.githubusercontent.com/25610170/220647738-852ab744-4dc7-4e59-8010-1eb8a20f44bd.png)

Clicking in any of the Splitted ports ( F9 or F10 ), gives nothing!
Any help will be greatly appreciated!
Andres

**Comment by MarianRychtecky** [comment]
Hi, this is Marian; we have entirely the same issue. I thought it was a problem with our data, but this looks like a bug. On top, our trace is working in A -> B direction, but not in B -> A direction.
